index.bs

```diff
@@ -22,26 +22,13 @@ Abstract:
   challenge by building on top of current and future web standards, to allow entities to authenticate
   within a Solid ecosystem.
 Status Text:
-  This section describes the status of this document at the time of its publication. Other documents
-  may supersede this document. A list of current W3C publications and the latest revision of this
-  technical report can be found in the [W3C technical reports index](https://www.w3.org/TR/) at
-  https://www.w3.org/TR/.
-
-  This document is produced from work by
-  the [Solid Community Group](https://www.w3.org/community/solid/). It is a draft document that may,
-  or may not, be officially published. It may be updated, replaced, or obsoleted by other documents at
-  any time. It is inappropriate to cite this document as anything other than work in progress. The
-  source code for this document is available at the following
-  URI: [https://github.com/solid/authentication-panel](https://github.com/solid/authentication-panel)
-
-  This document was published by the
-  [Solid Authentication Panel](https://github.com/solid/process/blob/master/panels.md#authentication)
-  as a First Draft.
-
-  [GitHub Issues](https://github.com/solid/authentication-panel/issues) are preferred for discussion
-  of this specification. Alternatively, you can send comments to our mailing list. Please send them to
-  [public-solid@w3.org](mailto:public-solid@w3.org)
-  ([archives](https://lists.w3.org/Archives/Public/public-solid/))
+  This section describes the status of this document at the time of its publication.
+
+  This document was published by the [Solid Community Group](https://www.w3.org/community/solid/) as an Editor’s Draft. The sections that have been incorporated have been reviewed following the [Solid process](https://github.com/solid/process). However, the information in this document is still subject to change. You are invited to [contribute](https://github.com/solid/solid-oidc/issues) any feedback, comments, or questions you might have.
+
+  Publication as an Editor’s Draft does not imply endorsement by the W3C Membership. This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.
```

Comment:
Do you intentionally use a fancy UTF-8 ’ (U+2018 | ‘ | e2 80 98 | LEFT SINGLE QUOTATION MARK) rather than the basic ASCII apostrophe?

```suggestion
  This document was published by the [Solid Community Group](https://www.w3.org/community/solid/) as an Editor's Draft. The sections that have been incorporated have been reviewed following the [Solid process](https://github.com/solid/process). However, the information in this document is still subject to change. You are invited to [contribute](https://github.com/solid/solid-oidc/issues) any feedback, comments, or questions you might have.

  Publication as an Editor's Draft does not imply endorsement by the W3C Membership. This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.
```

Reply:
Good question. @csarven do you have an opinion about that?

Reply:
It would also be nice to have shorter lines. I don't like strict rules about line length, but keeping them in the 80-120 char range makes reviews much easier

Reply:
I think each of those lines are a unit of data that should be versioned coherently. Adding line breakage rules makes everything messier and more difficult to work with in my experience.

I would say this really is a GitHub UI flaw.

The long lines are actually properly displayed in the code change suggestions because the `td` has `word-wrap: break-word;white-space: pre-wrap;` on it (if only they added the same style on `.blob-code-inner`... not sure where to flag that issue though).

Reply:
I mention this just because, in my experience, when I review PRs on a small device, it involves a lot of horizontal scrolling. So yes, it may be a GH UI issue, but it makes the review experience much harder. I wouldn't block the PR on that basis, but it's a real pain to review.

I would add that the rest of the document wraps lines at 80-120 chars, so this is a departure from the convention of the rest of the document.

Reply:
>Good question. @csarven do you have an opinion about that?

C'mon :) Not going to do minor editorial on the editors preference here. The punctuation police is off to lunch. Use whichever you prefer :)

>I would add that the rest of the document wraps lines at 80-120 chars, so this is a departure from the convention of the rest of the document.

+1 to consistency.

Reply:
I'll abide by the majority.
Amended with maximum line length.